lib/modules/home/components/body.dart

```diff
@@ -6,44 +6,63 @@ class Body extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final state = context.findAncestorStateOfType<_HomeViewImplState>()!;
-    return Screen(
-      body: Container(
-        color: Colors.black,
-        child: StreamsSelector0<bool>.value(
-          stream: state._isLoading,
-          builder: (_, isLoading, child) => Skeleton(
-            isShown: isLoading,
-            baseColor: Colors.white.withOpacity(0.12),
-            highlightColor: Colors.white.withOpacity(0.5),
-            borderRadius: BorderRadius.circular(16),
-            height: 18,
-            child: child!,
-          ),
-          child: Stack(
-            children: [
-              StreamsSelector0<List<SurveyInfo>>.value(
-                stream: state._surveys,
-                builder: (_, surveys, __) => CarouselSlider.builder(
-                  carouselController: state._pageController,
-                  itemCount: surveys.length,
-                  itemBuilder: (_, index, ___) => SlideItem(
-                    survey: surveys[index],
+    return StreamsSelector0<bool>.value(
+      stream: state.isProgressHUDShown,
```

Comment:
I see some inconsistency here:
- This value doesn't use `_underscoreName`. 
- The builder argument is not the same as its observed stream where the other places we do the same `isShown`

Reply:
@sleepylee 
> This value doesn't use _underscoreName.

Oh I will refactor this one after, we also discussed about this [here](https://github.com/markgravity/flutter-survey/pull/65#discussion_r642975276)

> The builder argument is not the same as its observed stream where the other places we do the same isShown

Updated! 🙏